tests/test-cases/Group1-Docker-Commands/1-02-Docker-Pull.robot

```diff
@@ -50,7 +50,7 @@ Pull an image with the full docker registry URL
     Wait Until Keyword Succeeds  5x  15 seconds  Pull image  registry.hub.docker.com/library/hello-world
 
 Pull an image with all tags
-    Wait Until Keyword Succeeds  5x  15 seconds  Pull image  --all-tags nginx
+    Wait Until Keyword Succeeds  5x  15 seconds  Pull image  --all-tags ${nginx}
```

Comment:
we don't have multiple nginx tags currently so doing this eliminates the purpose of the test